rhel8/profiles/ospp.profile

```diff
@@ -270,12 +258,6 @@ selections:
     ## RHEL 8 CCE-82943-2: Uninstall gssproxy Package
     - package_gssproxy_removed
 
-    ## RHEL 8 CCE-82939-0: Uninstall geolite2-city Package
-    - package_geolite2-city_removed
```

Comment:
IIRC, the geocity packages need to be explicitly removed to ensure the system does not contain geolocation capabilities. 

Is there another way to ensure this?

Reply:
I'm not aware of such requirement and I've been told their removal has no implication on OSPP claims. If you know of a specific SFR or other identifier, please do share.